__tests__/comments.test.js

```diff
@@ -0,0 +1,18 @@
+import getComments from '../src/module/__mocks__/comments';
+
+describe('Add tests for comments counter', () => {
+  it('Three Comments in a movie - Output: 3', async () => {
+    const commentsNumber = await getComments(53881); // Run the function
+    expect(commentsNumber.length).toEqual(3); // Make an assertion on the result
+  });
+
+  it('No Comments in a movie - Output: Undefined', async () => {
+    const commentsNumber = await getComments(538851); // Run the function
+    expect(commentsNumber.length).toEqual(undefined); // Make an assertion on the result
+  });
+
+  it('One Comments in a movie - Output: 1', async () => {
+    const commentsNumber = await getComments(19678); // Run the function
+    expect(commentsNumber.length).toEqual(1); // Make an assertion on the result
+  });
```

Comment:
- [ ] There is one test case that failed while running tests, So please fix that.

![image](https://user-images.githubusercontent.com/68796232/203790752-fb6a054b-78c5-4270-b974-63a6cd06aa84.png)